### PR TITLE
disable volta updates CPP-586

### PR DESCRIPTION
**Pull request by apaleslimghost**

### CODE REVIEW

File: renovate-config.json5 [hunk 1]
```diff
@@ -58,6 +58,11 @@
       ],
       dependencyDashboardApproval: true,
     },
+    {
+      // disable volta (i.e. pinned Node and npm version) updates
+      matchDepTypes: ['volta'],
```
Comment (ivomurrell) [inline renovate-config.json5:63]: **issue:** Doesn't `matchDepTypes` only encompass npm dependency types like `peerDependencies` and `optionalDependencies`?

Unfortunately, it looks like the `supportPolicy` field we were using to set node versions has recently been [removed](https://github.com/renovatebot/renovate/pull/11176) (and perhaps never worked seeing as [we don't use Travis](https://github.com/renovatebot/renovate/issues/11144)...?) According to the [docs](https://docs.renovatebot.com/node/), as far as I can see the only way to disable these renovations are by explicitly pinning the node version in the `engines` field in a repository's `package.json`, though hopefully I'm missing a way to disable it in the config too!

**Review by ivomurrell — CHANGES_REQUESTED** [review]
**issue:** looks like the pre-commit hook didn't run seeing as the change was made on GitHub (I've never though of that quirk before!) so the changes to the `.json5` have not been propagated to the `.json`.

**praise:** thanks for trying to rid us of this spam, though I think renovate might be making it a bit harder to do than it needs to be 😅

File: renovate-config.json5 (hunk 1 above)
Comment (apaleslimghost) [reply renovate-config.json5:63]: according to [this](https://docs.renovatebot.com/modules/manager/npm/#additional-information), "`depTypes`" includes `volta`. now, there's no actual option called `depTypes`, so i assume this refers to `matchDepTypes`.

it's kind of hilarious that `supportPolicy` was only for Travis. i could remove it in this PR?

Comment (ivomurrell) [reply renovate-config.json5:63]: ah yep I completely missed that part of the docs, that's a big relief!! (wish they'd linked to that from either the node page I linked or from the `matchDepTypes` definition...)

> it's kind of hilarious that supportPolicy was only for Travis. i could remove it in this PR?

probably makes sense, might avoid some confusion in the future

**Review by ivomurrell — APPROVED** [review]
*praise*: those renovate PRs were very annoying and I'm glad we're tidying up deprecated code too!

**Pull request merged**